services/core-api/app/api/mines/explosives_permit/resources/explosives_permit_document_type.py

```diff
@@ -70,12 +71,13 @@ def post(self, document_type_code):
 
             explosives_permit_guid = data['explosives_permit_guid']
             explosives_permit = ExplosivesPermit.find_by_explosives_permit_guid(explosives_permit_guid)
+            explosives_permit_amendment_id = data.get('explosives_permit_amendment_id', None)
```

Comment:
I think we might want to perform a check here to make sure the explosives_permit_amendment_id is actually for this explosives_permit. Just a precaution.

Reply:
A check for this has been added now.